tests/test_app.py

```diff
@@ -280,25 +283,52 @@ def test_version_p(self) -> None:
     def test_patch_save_json_p(self) -> None:
         target_path = str(SAMPLES_DIR / "password.patch")
         with tempfile.TemporaryDirectory() as tmp_dir:
-            json_filename = os.path.join(tmp_dir, "unittest_output.json")
+            json_filename = os.path.join(tmp_dir, f"{__name__}.json")
             _stdout, _stderr = self._m_credsweeper(
                 ["--diff_path", target_path, "--save-json", json_filename, "--log", "silence"])
-            assert os.path.exists(os.path.join(tmp_dir, "unittest_output_added.json"))
-            assert os.path.exists(os.path.join(tmp_dir, "unittest_output_deleted.json"))
+            assert os.path.exists(os.path.join(tmp_dir, f"{__name__}_added.json"))
+            assert os.path.exists(os.path.join(tmp_dir, f"{__name__}_deleted.json"))
 
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
     def test_patch_save_json_n(self) -> None:
         target_path = str(SAMPLES_DIR / "password.patch")
         _stdout, _stderr = self._m_credsweeper(["--diff_path", target_path, "--log", "silence"])
-        assert not os.path.exists(os.path.join(PROJECT_DIR, "unittest_output_added.json"))
-        assert not os.path.exists(os.path.join(PROJECT_DIR, "unittest_output_deleted.json"))
+        for root, dirs, files in os.walk(PROJECT_DIR):
+            self.assertIn('credsweeper', dirs)
+            for file in files:
+                self.assertFalse(file.endswith('.json'))
+            break
+
+    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+    def test_export_config_p(self) -> None:
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            json_filename = os.path.join(tmp_dir, f"{__name__}.json")
+            _stdout, _stderr = self._m_credsweeper(["--export_config", json_filename, "--log", "silence"])
+            assert os.path.exists(json_filename)
+
+    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+    def test_import_config_p(self) -> None:
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            json_filename = os.path.join(tmp_dir, f"{__name__}.json")
+            shutil.copyfile(CREDSWEEPER_DIR / "secret" / "config.json", json_filename)
+            _stdout, _stderr = self._m_credsweeper(["--config", json_filename, "--log", "silence"])
+            assert os.path.exists(json_filename)
```

Comment:
I think asserting `json_filename` is good, but it would be better to check the config file imported properly.
How about check `self.config` variable from `app.py` is initialized successfully?

Reply:
The test verifies whether current config might be loaded like custom config.
No direct way to check self.config due credsweeper is run in separated process.
I'll try to make more sense in the test

Reply:
@babenek 

I know it is difficult too.. Thank you for your trying :+1:

Reply:
Done. There are .py files are found with the extension adding in custom config